runtime/wasm/src/host.rs

```diff
@@ -127,6 +128,15 @@ impl RuntimeHost {
             "data_source" => config.data_source.name.clone(),
         ));
 
+        let api_version = Version::parse(&config.data_source.mapping.api_version)?;
+        if !VersionReq::parse("<= 0.0.2").unwrap().matches(&api_version) {
+            return Err(format_err!(
+                "This Graph Node only supports subgraph manifest specs <= 0.0.2, but subgraph `{}` uses `{}`",
```

Comment:
Now this message isn't correct anymore, it should `mapping API versions` instead of `subgraph manifest specs` 😉